ivy.el

```diff
@@ -3445,7 +3446,20 @@ In any Ivy completion session, the case folding starts with
   ;; Reset cache so that the candidate list updates.
   (setq ivy--old-re nil))
 
+(defmacro ivy--while-no-input (&rest body)
+  "Wrap BODY in `while-no-input' and respecting `non-essential'.
+Return the value if success running BODY."
```

Comment:
Nit: "Return result of last form in BODY if it finishes successfully."

Reply:
```suggestion
Return result of last form in BODY if it finished successfully."
```